zsl/utils/xml_to_json.py

```diff
@@ -10,13 +10,13 @@
 
```

Comment:
This utility is quite strange. I'd like to open a discussion if it should be kept here. I guess it fits better to the application.